Refunct/README.md

```diff
@@ -1,22 +1,11 @@
 # Refunct Autosplitter
-Readme by @firestack and @batedurgonnadie
+(Readme by firestack, BatedUrGonnaDie, Ero)
```

Comment:
You can keep the `@` to directly tag the GitHub accounts, making it easier to find them.

Reply:
Absolutely agree. This is my bad, I wasn't aware of this.

Reply:
Interesting, I would have imagined that to work in rendered markdown files just as it does for comments… Guess not then ¯\\\_(ツ)\_/¯

Reply:
I suppose one way to fix that is by linking to the user's page manually instead using \[link](example.com).

Reply:
At this point I think it would be better to remove the by mention at the top, and move it to a general contributors section after the instructions using manual links like you said @just-ero .  That way people can add themselves when they contribute (I can add prior committers) and they can link to whatever they want to be found on.